packages/angular_devkit/architect/BUILD

```diff
@@ -12,28 +12,19 @@ package(default_visibility = ["//visibility:public"])
 ts_library(
     name = "architect",
     srcs = glob(
-        ["src/**/*.ts"],
-        # Currently, this library is used only with the rollup plugin.
-        # To make it simpler for downstream repositories to compile this, we
-        # neither provide compile-time deps as an `npm_install` rule, nor do we
-        # expect the downstream repository to install @types/webpack[-*]
-        # So we exclude files that depend on webpack typings.
+        include = ["src/**/*.ts"],
         exclude = [
             "**/*_spec.ts",
             "**/*_spec_large.ts",
         ],
     ),
+    module_name = "@angular-devkit/architect",
+    module_root = "src/index.d.ts",
     data = glob(["**/*.json"]),
     deps = [
         "//packages/angular_devkit/core",
         "//packages/angular_devkit/core:node",
         "@rxjs",
         "@rxjs//operators",
     ],
-    tsconfig = "//:tsconfig.json",
-    module_name = "@angular-devkit/architect",
-    module_root = "src",
```

Comment:
why do we no longer need module root?

Reply:
I left it removed by mistake, back on I thought it wasn't needed. Added it back now.

Reply:
We do, but it's added above (before data).